### PR TITLE
binlogctl: add unit test

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 
improve test coverage

### What is changed and how it works?
add unit test, from 63% to 79%

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: binlogctl/meta_test.go [hunk 1]
```diff
@@ -48,3 +50,42 @@ func (s *metaSuite) TestSaveMeta(c *C) {
...
+	pd.Client
+	physical, logical int64
+	err               error
+}
+
+func (c dummyCli) GetTS(ctx context.Context) (int64, int64, error) {
+	return c.physical, c.logical, c.err
+}
+
+func NewFakePDClient([]string, pd.SecurityOption) (pd.Client, error) {
```
Comment (suzaku) [inline binlogctl/meta_test.go]: This doesn't have to be a public function?

File: binlogctl/nodes_test.go [hunk 2]
```diff
@@ -93,6 +100,13 @@ func (s *nodesSuite) TestUpdateNodeState(c *C) {
 	c.Assert(findNode, IsTrue)
 }
 
+func (s *testNodesSuite) TestCreateRegistry(c *C) {
+	urls := "127.0.0.1:2379"
+	registry, err := createRegistry(urls)
+	c.Assert(err, IsNil)
+	c.Assert(registry, NotNil)
```
Comment (suzaku) [inline binlogctl/nodes_test.go]: What else can we check about the returned `registry`?

Comment (WangXiangUSTC) [reply binlogctl/nodes_test.go]: add in https://github.com/pingcap/tidb-binlog/pull/604/commits/c4445f909466bab2b0432542146c4b1258943ab2

File: binlogctl/meta_test.go (hunk 1 above)
Comment (WangXiangUSTC) [reply binlogctl/meta_test.go]: done

**Comment by WangXiangUSTC** [comment]
@suzaku PTAL again

**Comment by suzaku** [comment]
LGTM

**Pull request merged**